### ISSUE TITLE
`pnpm i -g now` cli app not working

### ISSUE THREAD

**Issue by vjpr** [pnpm/pnpm#885]
I think now downloads and installs a prepackaged binary...maybe this is causing problems.

Also, the progress bar doesn't seem to show for pnpm. Maybe something to do with the way pnpm logs output.

# pnpm

```
$ pnpm i -g now

$ cat /Users/Vaughan/nvm/versions/node/v8.1.2/bin/now
NODE_PATH="/Users/Vaughan/nvm/versions/node/v8.1.2/pnpm-global/1/node_modules/.registry.npmjs.org/now/8.0.1/node_modules/now/node_modules:/Users/Vaughan/nvm/versions/node/v8.1.2/pnpm-global/1/node_modules/.registry.npmjs.org/now/8.0.1/node_modules:/Users/Vaughan/nvm/versions/node/v8.1.2/pnpm-global/1/node_modules/.registry.npmjs.org/now/node_modules:/Users/Vaughan/nvm/versions/node/v8.1.2/pnpm-global/1/node_modules/.registry.npmjs.org/node_modules:/Users/Vaughan/nvm/versions/node/v8.1.2/pnpm-global/1/node_modules:/Users/Vaughan/nvm/versions/node/v8.1.2/pnpm-global/node_modules:/Users/Vaughan/nvm/versions/node/v8.1.2/node_modules:/Users/Vaughan/nvm/versions/node/node_modules:/Users/Vaughan/nvm/versions/node_modules:/Users/Vaughan/nvm/node_modules:/Users/Vaughan/node_modules:/Users/node_modules:/node_modules:/Users/Vaughan/nvm/versions/node/v8.1.2/pnpm-global/1/node_modules/now/node_modules" "$basedir/../pnpm-global/1/node_modules/now/download/dist/now"   "$@"

$ now -v
/Users/Vaughan/nvm/versions/node/v8.1.2/bin/now: line 1: /../pnpm-global/1/node_modules/now/download/dist/now: No such file or directory
```

# npm

```
npm i -g now
npm WARN gentlyRm not removing /Users/Vaughan/nvm/versions/node/v8.1.2/bin/now as it wasn't installed by /Users/Vaughan/nvm/versions/node/v8.1.2/lib/node_modules/now
/Users/Vaughan/nvm/versions/node/v8.1.2/bin/now -> /Users/Vaughan/nvm/versions/node/v8.1.2/lib/node_modules/now/download/dist/now

> now@8.0.1 postinstall /Users/Vaughan/nvm/versions/node/v8.1.2/lib/node_modules/now
> node download/install.js

> For the source code, check out: https://github.com/zeit/now-cli

> Downloading now CLI 8.0.1 [====================] 100%

+ now@8.0.1
updated 1 package in 3.733s
```

### Additional information:

 - `node -v` prints: 8.1.2
 - Windows, OS X, or Linux?: mac

**Comment by zkochan** [comment]
now's bin file is not correctly linked into `.bin`. This is reproducible during both global and local installation.

**Comment by zkochan** [comment]
Fix published in [v1.14.8](https://github.com/pnpm/pnpm/releases/tag/v1.14.8)
